src/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet"/>
+  <title>To-Do List</title>
+</head>
+<body>
```

Comment:
Hello @JEFFKAY16 

Your project was definitely tested using Webpack. I think you shouldn't have unlinked your CSS and javascript files, and rather lint those files using the recommended linters. Apparently, your CSS file was improperly linked at the time of 
 review.

Reply:
@rloterh 

Thank you so much for follow-up. Based on my research, I was told that since we are using web-pack, we don't need to link a CSS file or JavaScript file to the html because the project is run on a server. What's the best way you'd suggest I handle this next time? I'd love to learn more for the sake of future projects and writing better code. 

Thank you once more. I really appreciate the feedback.